assets/queries/k8s/psp_allows_sharing_host_pid/query.rego

```diff
@@ -0,0 +1,16 @@
+package Cx
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    document.kind == "PodSecurityPolicy"
+    
+    document.spec.hostPID == true
+
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    "spec.hostPID",
```

Comment:
We filter by metadata
 "searchKey":        sprintf("metadata.name=%s.spec.hostPID", [metadata.name]),